addons/Drawing/src/presenter.js

```diff
@@ -81,7 +86,10 @@ function AddonDrawing_create() {
 
     presenter.points = [];
     presenter.pointerPosition = {x: 0, y: 0};
-    presenter.isStarted = false;
+    presenter.shouldUpdateState = false;
+    presenter.isModified = false;
+    let shouldSendModifiedEventOnLeave = false;
+    let blockNextSendOfModifiedEvent = false;
```

Comment:
teraz dopiero to zauważyłem. Mój błąd 👎 
Nazwa zmiennej nie powinna nazywać się jak nazwa funkcji. Przy okazji proszę o poprawienie na lepszą nazwę

Reply:
Dziękuję :+1